.gitignore

```diff
@@ -1 +0,0 @@
-node_modules/
```

Comment:
@andmiguedev why was this deleted?

Reply:
I run npm install in a folder that didn't have package.json by mistake. If you see it is not a valid .gitignore for React

Reply:
if you checkout to the branch, there should be a valid gitignore under server-api